solana/program_v2/programs/gateway_v2/src/instructions/admin/close_network.rs

```diff
@@ -11,8 +11,9 @@ pub struct CloseNetworkAccount<'info> {
     #[account(
         mut,
         close = destination,
-        seeds = [NETWORK_SEED, network.initial_authority.key().as_ref()],
-        bump = network.signer_bump,
+        // TODO? Do I need to update this to include the network index?
```

Comment:
Yeah, see:
https://github.com/identity-com/on-chain-identity-gateway/blob/0a120a7427d285847879d339300381878f480b1e/solana/program_v2/programs/gateway_v2/src/instructions/gatekeeper/issue_pass.rs#L31

https://github.com/identity-com/on-chain-identity-gateway/blob/0a120a7427d285847879d339300381878f480b1e/solana/program_v2/src/GatekeeperService.ts#L84

Not zure if it's best, but it works

Reply:
👍